proposed/2021/BulkRestoreCoordination.md

```diff
@@ -0,0 +1,170 @@
+# Bulk restore coordination
+
+- [Nikolche Kolev](https://github.com/nkolev92)
+- Start Date 2021-05-03
+- GitHub Issue [10678](https://github.com/NuGet/Home/issues/10678)
+- GitHub PR [4058](https://github.com/NuGet/NuGet.Client/pull/4058)
+
+## Summary
+
+<!-- One-paragraph description of the proposal. -->
+When a solution with SDK based projects is loaded, NuGet uses a sliding window to coordinate the number of restores being run to avoid extra restore/design time build loops.

```

Comment:
By `sliding window` means `sliding window of time frame` in below?

https://github.com/NuGet/NuGet.Client/blob/1ef0444b1ff1ca79ebde7a6e3a717a4ff74b1832/src/NuGet.Clients/NuGet.SolutionRestoreManager/SolutionRestoreWorker.cs#L505

Reply:
yes, that line + the 3 lines before.